app/hotels/src/singleHotel/roomPicker/SelectButton.js

```diff
@@ -0,0 +1,64 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import {
+  Price,
+  StyleSheet,
+  Touchable,
+  ButtonTitle,
+} from '@kiwicom/mobile-shared';
+import { Translation } from '@kiwicom/mobile-localization';
+import { defaultTokens } from '@kiwicom/mobile-orbit';
+
+type Props = {|
+  +price: number | null,
+  +currency: string | null,
+  +increment: () => void,
+|};
+
+export default function SelectButton({ increment, price, currency }: Props) {
+  return (
+    <Touchable
+      style={[
+        styles.container,
+        styles.borderRadiusLeft,
```

Comment:
I cannot see this in the styles object. (Idem for `styles.borderRadiusRight`)

Reply:
Yes, this is the result of copy/paste, I deleted them now